Caboodle/FileSystem/FileSystem.uwp.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.IO;
+using System.Threading.Tasks;
+using Windows.ApplicationModel;
+using Windows.Storage;
+
+namespace Microsoft.Caboodle
+{
+    public static partial class FileSystem
+    {
+        public static string CacheDirectory
+            => ApplicationData.Current.LocalCacheFolder.Path;
+
+        public static string AppDataDirectory
+            => ApplicationData.Current.LocalFolder.Path;
+
+        public static string UserDataDirectory
+            => ApplicationData.Current.LocalFolder.Path;
```

Comment:
Is this going to cause any chaos that `AppDataDirectory` and `UserDataDirectory` are the same path?  Like if a user was to `Directory.Delete(AppDataDirectory)` without understanding that also wipes `UserDataDirectory` ?